bats/tests/helpers/defaults.bash

```diff
@@ -74,6 +74,33 @@ if using_networking_tunnel && ! is_windows; then
     fatal "RD_USE_NETWORKING_TUNNEL only works on Windows"
 fi
 
+########################################################################
+: "${RD_MOUNT_TYPE:=reverse-sshfs}"
+
+validate_enum RD_MOUNT_TYPE reverse-sshfs 9p virtiofs
+
+if [ "$RD_MOUNT_TYPE" = "virtiofs" ] && ! using_vz_emulation; then
+    fatal "RD_MOUNT_TYPE=virtiofs only works with VZ emulation"
+fi
+
+########################################################################
+: "${RD_9P_CACHE_MODE:=mmap}"
```

Comment:
Please, add fatal function in case the 9p dependent variables are set, but `"$RD_MOUNT_TYPE" != 9p`

Reply:
I don't want to do this; the settings are just being ignored when the mount type is not `9p`. This makes it easier to set defaults that only apply when using `9p`.

Setting a variable that is being ignored is different from setting a variable to a value that does not work.